server/src/workflow-management/classes/Generator.ts

```
@@ -1027,77 +1111,107 @@ export class WorkflowGenerator {
+          pair.what.splice(currentIndex, 1);
+          continue;
         }
-      });
+  
+        currentIndex++;
+      }
     }
+  
+    for (const [selector, state] of inputStates.entries()) {
+      if (state.value) {
+        for (let i = workflow.workflow.length - 1; i >= 0; i--) {
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)